src/Media_Command.php

```diff
@@ -196,6 +196,7 @@ public function regenerate( $args, $assoc_args = array() ) {
 	 *
 	 * [--skip-copy]
 	 * : If set, media files (local only) are imported to the library but not moved on disk.
+   * File names will not be run through wp_unique_filename() with this set.
```

Comment:
```suggestion
	* File names will not be run through wp_unique_filename() with this set.
```

You need to use tabs instead of spaces 🙂